bin/optimize-schemas.ts

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ts-node-transpile-only
+
+import { strict as assert } from "assert";
+import fs from "fs";
+import { JSONSchema7 } from "json-schema";
```

Comment:
just a nitpick, but maybe we should start using `import type` when importing types wherever possible

Reply:
Sure - theres no particular advantage to them in this sort of context, but no harm in it.